common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdConnectPromptParameter.java

```diff
@@ -35,6 +35,11 @@
     /**
      * No prompt parameter will be injected into the request.
      */
+    NOT_SENT,
```

Comment:
`NOT_SENT` seems a bit strange, how about `NOT_SET` , `NULL` . Shane suggested `UNSET` when we were discussing this as well

Reply:
@kreedula sure UNSET would be good!